app/layout.tsx

```diff
@@ -62,13 +62,13 @@ export default function RootLayout({
       <body>
         <Suspense>
           <Providers>
-            <div className="flex flex-col md:flex-row">
+            <div className="flex flex-col md:flex-row h-full">
```

Comment:
why are you using all 3 of these?
- here: `h-full`
- global.css: firstOfType `height: 100%;`
- global.css: `body: height: 100%;`

Is there a reason for all 3? if no, which, specific, styling is actually making the change you're trying to make?